packages/app-store/zapier/api/subscriptions/addSubscription.ts

```diff
@@ -38,7 +39,14 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
       //schedule job for already existing bookings
       const bookings = await prisma.booking.findMany({
         where: {
-          userId: validKey.userId,
+          AND: [
+            { userId: validKey.userId },
+            {
+              eventType: {
+                teamId: validKey.teamId,
+              },
+            },
+          ],
```

Comment:
I think you can omit AND sentence and you still get the same result.

Reply:
Shouldn't be one OR another?